src/components/GithubCard.js

```diff
@@ -0,0 +1,48 @@
+import {Github, Globe2, JournalCode, PeopleFill, Twitter} from "react-bootstrap-icons";
+import {Card} from "react-bootstrap";
+import {useState} from "react";
+
+function GithubCard(props) {
```

Comment:
This component is THICC. 

Honestly, you could've probably broken it up into a couple more components, but this still works so well done.